Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from compechem import System
from compechem import XtbInput
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mImportError[0m                               Traceback (most recent call last)
Input [0;32mIn [1][0m, in [0;36m<cell line: 1>[0;34m()[0m
[0;32m----> 1[0m [38;5;28;01mfrom[39;00m [38;5;21;01mcompechem[39;00m [38;5;28;01mimport[39;00m System
[1;32m      2[0m [38;5;28;01mfrom[39;00m [38;5;21;01mcompechem[39;00m [38;5;28;01mimport[39;00m XtbInput

File [0;32m/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/compechem/__init__.py:5[0m, in [0;36m<module>[0;34m[0m
[1;32m      2[0m [38;5;28;01mfrom[39;00m [38;5;21;01mcompechem[39;00m[38;5;21;01m.[39;00m[38;5;21;01mwrappers[39;00m[38;5;21;01m.[39;00m[38;5;21;01morca[39;00m [38;5;28;01mimport[39;00m OrcaInput, M06, CCSD, r2SCAN
[1;32m      3[0m [38;5;28;01mfrom[39;00m [38;5;21;01mcompechem[39;00m[38;5;21;01m.[39;00m[38;5;21;01mwrappers[39;00m[38;5;21;01m.[39;00m[38;5;21;01mxtb[39;00m [38;5;28;01mimport[39;00m XtbInput
[0;32m----> 5[0m [38;5;28;01mfrom[39;00m [38;5;21;01mcompechem[39;00m[38;5;21;01m.[39;00m[38;5;21;01mwrappers[39;00m [38;5;28;01mimport[39;00m crest
[1;32m      7[0m [38;5;28;01mfrom[39;00m [38;5;21;01mcompechem[39;00m[38;5;21;01m.[39;00m[38;5;21;01mwrappers[39;00m [38;5;28;01mimport[39;00m packmol
[1;32m      9[0m [38;5;28;01mfrom[39;00m [38;5;21;01mcompechem[39;00m[38;5;21;01m.[39;00m[38;5;21;01msystems[39;00m [38;5;28;01mimport[39;00m System, Ensemble, MDTrajectory

File [0;32m/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/compechem/wrappers/crest.py:4[0m, in [0;36m<module>[0;34m[0m
[1;32m      2[0m [38;5;28;01mfrom[39;00m [38;5;21;01mtempfile[39;00m [38;5;28;01mimport[39;00m mkdtemp
[1;32m      3[0m [38;5;28;01mfrom[39;00m [38;5;21;01mcompechem[39;00m[38;5;21;01m.[39;00m[38;5;21;01mconfig[39;00m [38;5;28;01mimport[39;00m get_ncores
[0;32m----> 4[0m [38;5;28;01mfrom[39;00m [38;5;21;01mcompechem[39;00m [38;5;28;01mimport[39;00m System
[1;32m      5[0m [38;5;28;01mfrom[39;00m [38;5;21;01mcompechem[39;00m[38;5;21;01m.[39;00m[38;5;21;01mtools[39;00m [38;5;28;01mimport[39;00m split_multixyz
[1;32m      6[0m [38;5;28;01mfrom[39;00m [38;5;21;01mcompechem[39;00m[38;5;21;01m.[39;00m[38;5;21;01mtools[39;00m [38;5;28;01mimport[39;00m cyclization_check

[0;31mImportError[0m: cannot import name 'System' from partially initialized module 'compechem' (most likely due to a circular import) (/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/compechem/__init__.py)
ImportError: cannot import name 'System' from partially initialized module 'compechem' (most likely due to a circular import) (/opt/hostedtoolcache/Python/3.8.13/x64/lib/python3.8/site-packages/compechem/__init__.py)

